test/test_meta.py

```diff
@@ -980,6 +993,17 @@ def __torch_dispatch__(self, func, types, args=(), kwargs=None):
             run_symbolic_meta=self.symbolic_meta,
         )
 
+def _handle_foreach_kwargs(func, args, kwargs):
+    if 'zero_size' in kwargs:
+        kwargs.pop('zero_size')
+    if 'disable_fastpath' in kwargs:
+        kwargs.pop('disable_fastpath')
+    if func in (torch._foreach_addcmul, torch._foreach_addcdiv):
+        if 'values' in kwargs:
+            values = kwargs.pop("values")
+            if values is not None:
+                args.append(values)
```

Comment:
foreach OpInfos use aux kwargs that other foreach tests also remove/pre-process.

Also: foreach metas were not tested. We only have tests with concrete values and inductor tests.